modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPhpCodegen.java

```diff
@@ -380,10 +417,6 @@ public String getSchemaType(Schema p) {
         return toModelName(type);
     }
 
-    public String getInvokerPackage() {
-        return invokerPackage;
-    }
-
```

Comment:
this shouldn't be removed as it will result in compilation errors. i will add it back

Reply:
fixed via https://github.com/OpenAPITools/openapi-generator/pull/16078